templates/prometheus.yaml

```diff
@@ -26,7 +26,7 @@ spec:
           - name: fluentbit-service-monitor
```

Comment:
Can you change the name `kubeaddons-service-monitor-api-v1-metrics-prometheus

Reply:
done.